src/pages/HomePage.jsx

```diff
@@ -0,0 +1,75 @@
+import { useSelector, useDispatch } from 'react-redux';
+import React, { useEffect, useState } from 'react';
+import { IoMdMic } from 'react-icons/io';
+import { FiSettings } from 'react-icons/fi';
+import { AiOutlineLeft } from 'react-icons/ai';
+import { fetchCountriesData } from '../Redux/Countries/Countries';
+import './HomePage.scss';
+import { getMapUrl } from '../Utilities/Map';
+import CountriesList from '../components/CountriesList';
+
+const HomePage = () => {
+  const dispatch = useDispatch();
+  const { status, data } = useSelector((state) => state.Countries);
+  const [search, setSearch] = useState('');
+
+  useEffect(() => {
+    if (status === 'Not Fetched') {
+      dispatch(fetchCountriesData());
+    }
+  }, []);
+
+  const filterData = (e) => {
+    const string = e.target.value.toLowerCase();
+    const info = data.filter(((country) => (country.id === string)));
+    if (info !== 0) {
+      setSearch(info);
+    } else {
+      setSearch(data);
+    }
+  };
+
+  const worldMap = getMapUrl('world');
+
+  return (
+    <>
+      <div className="header-container">
+        <div className="ouline-left">
+          <AiOutlineLeft />
+          <h3>2022</h3>
+        </div>
+        <h4 className="center-title" data-testid="title">Most Views</h4>
+        <div className="header-buttons">
+          <button type="button" className="microphone" label="microphone"><IoMdMic /></button>
+          <button type="button" className="settings" label="Settings" title="settings"><FiSettings /></button>
+        </div>
+      </div>
+      { status === 'success' && (
+      <>
+        <div className="firts-card">
+          <div className="map-container">
+            <img src={worldMap} alt="World Map" className="map" />
+          </div>
+          <div>
+            <h1>DAILY STATISTICS</h1>
+            <h2>{data[32].name.toUpperCase()}</h2>
+            <p>
+              {`${data[32].confirm} new infections`}
+            </p>
+            <input className="input" type="text" name="search" onChange={filterData} placeholder="Country Name" />
+          </div>
+        </div>
+        <div className="stast">
+          STATS BY COUNTRY
+        </div>
+        <div className="countries-container">
+          { search === ''
+            ? (<CountriesList countries={data} />) : (<CountriesList countries={search} />)}
```

Comment:
- [ ] I would recommend you follow the design template as provided in the guidelines. You have about 3 combinations of colors on the Homepage whereas what we have on the template is two alternating thick and lighter colors. Kindly fix this.

  Your design | Behance Design
  :---:|:---:
  ![alexr-alternating](https://user-images.githubusercontent.com/36542986/169161467-83e02fa5-b7c3-4150-8400-fb5e68706e54.jpg) | ![jonathas-orig-home](https://user-images.githubusercontent.com/36542986/169161698-0197b02a-5c60-4a27-b062-03348cd8585c.jpg)

- [ ] I would recommend you fix your Homepage design for listing of the countries. It has some white space which I would recommend you resolve, just as have done with the detail page.

  ![alexr-spaces](https://user-images.githubusercontent.com/36542986/169162186-2578c7c8-b0b9-40b0-a4c2-f4e196a8e3ca.PNG)